packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -339,6 +339,9 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// The [actions] become the trailing component of the [NavigationToolbar] built
   /// by this widget. The height of each action is constrained to be no bigger
   /// than the [toolbarHeight].
+  ///
+  /// To avoid that the last action is covered by the debug banner, set the
```

Comment:
To avoid having the last action covered by the debug banner, you may want to set the ...